map_with_sidebar.css

```diff
@@ -1,22 +1,77 @@
-/* This is for a page with a fullscreen map and a collapsible left sidebar */
+/* This is for a page with a map, collapsable sidebar, and collapsable bottom form area. */
 
-/* The map */
+/* Defaults */
+* {
+  box-sizing: border-box;
+}
 body {
-  margin: 0;
-  padding: 0;
+  font: 16px Helvetica Nueu,Arial,Helvetica,sans-serif;
 }
 
-#map {
+/* The application layout */
+body {
+  margin: 0;
+  padding: 0;
   position: absolute;
   top: 0;
+  left: 0;
+  right: 0;
   bottom: 0;
-  width: 100%;
+
+  overflow: hidden;
+
+  /* Two columns, the sidebar on the left, and the main content on the right. */
+  display: flex;
+  flex-flow: row nowrap;
+}
+
+body > aside {
+  position: relative; /* for children who want to position:absolute */
+
+  flex: 0;
+  min-width: 25rem; /* use min/max-width because we're in a flex containter. */
+  max-width: 25rem;
+}
+.sidebar-content {
+  width: 25em; /* Keep the size constant, so the map overlaps it when it collapses. */
+
+  max-height: 100%;
+  overflow-y: auto;
+}
+body > aside.collapsed {
+  min-width: 1.5rem;
```

Comment:
![Screenshot from 2022-10-24 14-33-28](https://user-images.githubusercontent.com/1664407/197538193-61b6bc9f-34fd-4191-9395-fe5785da76be.png)
This looks a bit weird to me; I might adjust this later to totally hide it

Reply:
Yes, it does look a bit janky. I was going for a nice hint that there is content there to be revealed, but it needs a bit more finesse to look intentional. Maybe a big drop shadow would make it look deliberate. Maybe `.sidebar-content { position: absolute; right: 0; }`, so that the content is hanging off the edge of the screen and just poking in would do the trick.

Or `min-width: 0`, shift the toggle a bit to the right and call it a day :)